### PR TITLE
Update Realtek Ameba Lite release SDK libraries

### PR DESCRIPTION
**Pull request by kaikaikaime**
Hi,

This is the SDK release version of 10.0d containing the latest libraries to date.

Regards,
Kai

### CODE REVIEW

File: os/board/rtl8720e/src/component/mbed/targets/hal/rtl8720e/us_ticker.c [hunk 1]
```diff
@@ -63,7 +63,7 @@ uint32_t us_ticker_read(void)
 	uint64_t us_tick;
 
 	tick_cnt = SYSTIMER_TickGet(); //up counter
-	us_tick = (uint64_t)(tick_cnt * (1000000 / 32768));
```
Comment (sunghan-chang) [inline os/board/rtl8720e/src/component/mbed/targets/hal/rtl8720e/us_ticker.c:66]: Due to SVACE issue (overflow), we pushed the change like this. Please confirm this does not cause it again.

File: os/board/rtl8720e/src/component/network/ping/ping_test.c [hunk 2]
```diff
@@ -3,6 +3,7 @@
 #include "main.h"
 
 #include <lwipconf.h>
+#include <lwip_netconf.h>
```
Comment (sunghan-chang) [inline os/board/rtl8720e/src/component/network/ping/ping_test.c:6]: Please revert a file permission to 644.
There are many files which are changed to 755. Please check all of them.

**Review by sunghan-chang — COMMENTED** [review]
1. We can't accept commit format you used. Please follow our style.
2. Previous code has many SVACE issues. Could you confirm new one does not have?
3. Please check build warnings as well.
4. What is the 10.0d version? Realtek internal version?

**Review by sunghan-chang — COMMENTED** [review]
1. Some c, h and bin files are still 755. Please change them to 644 permission.
2. sh file should be 755, not 644 because of execution.
3. Could you clean up the commit as 2 or 3 commits?
    1. code change in build folder
    2. code change in os folder (if possible, you can split them as 3 commits for arch, board and se)
4. Please confirm all of svace and build warnings are resolved.

**Comment by sunghan-chang** [comment]
5. There are many conflicts. Please check it as well.

**Comment by kaikaikaime** [comment]
Hi,
I have made the changes above
1. The commit has been squash to a single commit and commit formatting has been changed
2. User permissions have been fixed
3. The updated SDK libraries came from an aggregated commits to one single patch and were tied with to one another. Therefore splitting them to several commits may not be advisable, however if you insist I will try.

Regards,
Kai

**Comment by kaikaikaime** [comment]
Hi All, I did a forced push and inevitably pull request closed by itself. 
I reopened a new PR here: [here](https://github.com/Samsung/TizenRT/pull/5701)

**Pull request closed without merging**